src/web/Content/client/dumpling.py

```diff
@@ -944,6 +953,24 @@ def _add_key_if_not_exists(dictProp, key, val):
         if not key in dictProp:
             dictProp[key] = val
 
+    @staticmethod
+    def _create_hang_dump(pid, outpath, debuggerpath):
+        osStr = platform.system().lower()
+        command = ""
```

Comment:
We should do some kind of validation that the path stored in `debuggerpath` actually exists